mlflow/R/mlflow/R/tracking-rest.R

```diff
@@ -57,8 +69,25 @@ mlflow_rest <- function( ..., client, query = NULL, data = NULL, verb = "GET", v
     ),
     stop("Verb '", verb, "' is unsupported.")
   )
-  if (identical(response$status_code, 500L)) {
-    stop(xml2::as_list(content(response))$html$body$p[[1]])
+  if (response$status_code != 200) {
+    message_body <- tryCatch(
+      paste(content(response, "parsed", type = "application/json"), collapse = "; "),
+      error = function(...) {
+        tryParseResponseAsText(response)
+      },
+      warning = function(...) {
```

Comment:
When does this throw a warning?

Reply:
tbh I am not sure. I just wanted to cover all cases, this is what I found online as pattern for try/catch. 

Does this make sense or should I remove the warning section?

Reply:
Let's remove the warning handlers.

Reply:
Done.